lib/windows/winlean.nim

```diff
@@ -517,7 +517,7 @@ type
     ai_family*: cint        ## Address family of socket.
     ai_socktype*: cint      ## Socket type.
     ai_protocol*: cint      ## Protocol of socket.
-    ai_addrlen*: csize_t        ## Length of socket address.
+    ai_addrlen*: csize        ## Length of socket address.
```

Comment:
``csize_t`` is correct.

Reply:
Oh woops, not sure how that snuck in there

Reply:
```suggestion
    ai_addrlen*: csize_t        ## Length of socket address.
```